src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -97,5 +90,39 @@ public void Load(NodeSettings nodeSettings)
 
             this.callback?.Invoke(this);
         }
+
+        /// <summary>
+        /// Determines the default API port.
+        /// </summary>
+        /// <param name="network">The network to use.</param>
+        /// <returns>The default API port.</returns>
+        private static int DefaultPort(Network network)
+        {
+            int port;
+            if (network.IsBitcoin())
+            {
+                port = network.IsTest() ? TestBitcoinApiPort : DefaultBitcoinApiPort;
+            }
+            else
+            {
+                port = network.IsTest() ? TestStratisApiPort : DefaultStratisApiPort;
+            }
+
+            return port;
+        }
+
+        /// <summary>Prints the help information on how to configure the rpc settings to the logger.</summary>
+        /// <param name="network">The network to use.</param>
+        public static void PrintHelp(Network network)
+        {
+            var defaults = NodeSettings.Default();
```

Comment:
you can move the `var defaults` to the bottom, closer to where it's actually being used.

Reply:
Done.